Azure-Samples/communication-services-ios-calling-hero
# [Feature] Join call screen layout and validation

## Purpose
Building out the join call screen from the new design using fluent UI components.

## Does this introduce a breaking change?
```
[x] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?
```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code
* Ensure you have set up AAD
* Go to join call screen and observe new design and form validation

## What to Check
Verify that the following are valid
* Form validation
* Error pop-ups

## Other Information
